Vagrantfile

```diff
@@ -875,3 +875,40 @@ Vagrant.configure("2") do |config|
     trigger.on_error = :continue
   end
 end
+
+class VVVScripts < Vagrant.plugin(2, :command)
+  def self.synopsis
+    "Execute script from /config/homebin/"
+  end
+
+  def execute
+    with_target_vms(nil, single_target: true) do |vm|
+      @env.ui.output "#{$yellow}Executing #{$red}#{ARGV[1]}#{$creset}\n"
+      vm.action(:ssh_run, ssh_run_command: "/srv/config/homebin/#{ARGV[1]}" )
+    end
+  end
+end
+
+class VVVCommand < Vagrant.plugin(2, :command)
+  def self.synopsis
+    "Execute custom command inside site folder"
+  end
+
+  def execute
+    with_target_vms(nil, single_target: true) do |vm|
+      @env.ui.output "#{$yellow}Executing in #{$red}#{ARGV[1]}#{$yellow}: #{$red}#{ARGV[2]}#{$creset}\n"
+      vm.action(:ssh_run, ssh_run_command: "cd /srv/www/#{ARGV[1]}/public_html; #{ARGV[2]}")
```

Comment:
Is there a case for people wanting to run a command above public_html?

Reply:
this is the example of the wordpress-develop template, where you need to execute `grunt watch` on the vm instead of the host for issues as the ticket say.
I think that can be helpful also for plugin development with a switch to the folder in case there are symlinks required inside the VM.

Reply:
This won't work with sites that use the `vm_dir` variable